alpine-configure.sh

```diff
@@ -0,0 +1,9 @@
+./configure \
+  CFLAGS="-D__ALPINE__" \
```

Comment:
@olehermanse @vpodzime what do you think about just submitting a PR to the alpine packaging repo which adds ```CFLAGS="$CFLAGS -D__BUSYBOX__"```

I have tested this locally on an alpine laptop and seems to fix ps parsing nicely.

I will run tests next to check, there may be a lot of various commands changed to busybox versions in the C code because of this change.

Reply:
> a PR to the alpine packaging repo which adds CFLAGS="$CFLAGS -D__BUSYBOX__"

Where would it be added? And what about people compiling on Alpine Linux on their own? I'd prefer this would be detected in `configure.ac` auto-magically. And in there, it should add `-D__BUSYBOX__` to `CPPFLAGS` not `CFLAGS`.

Reply:
right, make ./autogen.sh work :+1: agreed.